build/Targets/Version.targets

```diff
@@ -0,0 +1,38 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <AssemblyVersion>1.0.0</AssemblyVersion>
```

Comment:
This isn't correct. AppDesigner/Editors needs to be 15.0.0.0.

How is this file different from VSL.Versions.targets? Should we remove the later?